testing/trino-tests/src/test/java/io/trino/tests/TestTableFunctionInvocation.java

```diff
@@ -83,6 +83,19 @@ protected Session getSession()
         return session;
     }
 
+    @Test
+    public void testBigintDefaultArgument()
+    {
+        // using assertThat() because assertQuery() does not verify the returned schema for empty result
```

Comment:
slightly related: https://github.com/trinodb/trino/pull/13032

`assertQuery` doesn't verify schema of the result _at all_, so please remove the comment.